pkg/sensors/tracing/killer.go

```diff
@@ -169,18 +169,32 @@ func (kh *killerHandler) createKillerSensor(
 				return nil, fmt.Errorf("Error list '%s' not found", listName)
 			}
 
-			if !isSyscallListType(list.Type) {
-				return nil, fmt.Errorf("Error list '%s' is not syscall type", listName)
-			}
 			kh.syscallsSyms = append(kh.syscallsSyms, list.Values...)
 			continue
 		}
 
-		pfxSym, err := arch.AddSyscallPrefix(sym)
-		if err != nil {
-			return nil, err
+		kh.syscallsSyms = append(kh.syscallsSyms, sym)
+	}
+
+	var err error
+
+	// fix syscalls
+	for idx, sym := range kh.syscallsSyms {
```

Comment:
```
  killers:
  - syscalls:
    - "security_file_permission"
```

I think we should rename `syscalls` to `calls` (or similar) since it's not just syscalls anymore.

Reply:
ok